### PR TITLE
Enhancing BPEL process visualization diagrams

### PR DESCRIPTION
**Pull request by NatashaWso2**

### CODE REVIEW

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java [hunk 1]
```diff
@@ -423,9 +428,9 @@ public Element getSVGString(SVGDocument doc) {
         Element group = null;
         group = doc.createElementNS("http://www.w3.org/2000/svg", "g");
         group.setAttributeNS(null, "id", getLayerId());
-        if (isAddOpacity()) {
+       /* if (isAddOpacity()) {
```
Comment (vinok88) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java]: Remove these commented lines

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ProcessImpl.java [hunk 2]
```diff
@@ -289,14 +289,17 @@ private Element drawLink(SVGDocument doc, int startX, int startY, int endX, int
         path.setAttributeNS(null, "id", id);
         path.setAttributeNS(null, "style", getLinkArrowStyle());
         path.setAttributeNS("xlink", "title", linkName);
-        //path.setAttributeNS(null, "style", "opacity:" + getIconOpacity());
-        //path.setAttributeNS(null, "onmouseover", "this.style.opacity=1;this.filters.alpha.opacity=100");
-        //path.setAttributeNS(null, "onmouseout", "this.style.opacity=" + getIconOpacity() + ";this.filters.alpha.opacity=25");
 
+        /* Removing opacity on mouseover and mouseout
```
Comment (vinok88) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ProcessImpl.java]: Remove these lines completely. Don't keep commented lines.

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java (hunk 1 above)
Comment (NatashaWso2) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ActivityImpl.java]: Fixed with commit 9e33b67

File: components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ProcessImpl.java (hunk 2 above)
Comment (NatashaWso2) [inline components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/ProcessImpl.java]: Fixed with commit 9e33b67

**Pull request merged**